BEACON-V2-draft4-Model/analyses/endpoints.json

```diff
@@ -130,7 +130,7 @@
         }
       }
     },
-    "/analyses/{id}/variants_in_sample": {
+    "/analyses/{id}/g_variants": {
```

Comment:
I'm still not a fan of the single letter "prefix" here & would rather just use either `variants` or `genomic_variations`. But this is a general point for the endpoint & therefore correct here, as long as this isn't changed everywhere :-/